test/maxwell/components/MaterialTest.cpp

```diff
@@ -10,10 +10,10 @@ class MaterialTest : public ::testing::Test {
 
 TEST_F(MaterialTest, impedanceAndConductance)
 {
-	Material mat1(1.0, 2.0);
-	Material mat2(100.0, 1.0);
-	Material mat3(10.0, 20.0);
-	Material mat4(9.0, 30.0);
+	Material mat1(1.0  , 2.0 , 0.0);
```

Comment:
Rather than having to change this everywhere to include conductivuty, you could simply assign a default value of zero in the ctor declaration/definition.